OneCommunityGlobal/HighestGoodNetworkApp
# Masasa - Rearranged permissions and indented sub_permissions

# Description
This pull request implements the requirements as specified by the screenshot below:
<img width="371" alt="Screenshot 2023-08-15 at 3 10 24 PM" src="https://github.com/OneCommunityGlobal/HighestGoodNetworkApp/assets/73863365/4ab99fbc-fbb9-4804-affa-7a5d5041f087">
<img width="731" alt="Screenshot 2023-08-21 at 12 51 55 PM" src="https://github.com/OneCommunityGlobal/HighestGoodNetworkApp/assets/73863365/fdfabc2e-00a9-43e4-98f4-f9d7f870452b">

The goal is to rearrange the permissions such that they are ordered according to how they are organized in this [spreadsheet](https://docs.google.com/spreadsheets/d/1TfiJY9OLDZuyP2UgF0C1wI9tOSLKbS9MiJFH6n1Dsao/edit#gid=0)

Then after ordering the should be formatted to have the permissions in white being the main ones while the ones in blue become indented to show that they are subordinate to the ones in the white row.

## Related PRS (if any):
it is not related to any back end PR
…

## Main changes explained:
- rearranged the order of permissions in UserRoleTab
- added new logic and formatting in rolePermissions …
…

## How to test:
1. check into current branch
2. do `npm install` and `...` to run this PR locally
3. log as owner user
4. go to otherlinks→ Permission Management→ choose any Role→…
5. verify that it looks like the screenshot

## Screenshots or videos of changes:

<img width="873" alt="Screenshot 2023-08-17 at 5 22 16 PM" src="https://github.com/OneCommunityGlobal/HighestGoodNetworkApp/assets/73863365/4564f677-b21a-4e6d-b3eb-ed09e09bc613">

## Note:
Include the information the reviewers need to know.
